docs/adr/0018-use-tailwind-with-custom-scss.md

```diff
@@ -0,0 +1,136 @@
+  - Note: class name conventions for the custom SCSS will be declared when the framework configuration is setup, as we'd like them to make sense together. Examples will be provided in the `fxa-settings` README at this time as well.
+
+## Pros and Cons of the Options
+
+### Option A - Use an existing UI library or framework
+
+Use an existing third-party UI library or framework that provides prebuilt components such as cards, form elements, loaders, various grid and section layouts, and modals, which we would then adjust to suit the needs of our design.
+
+This could include generally "full purpose" UI libraries and frameworks such as [Semantic UI](https://semantic-ui.com/), [Tailwind UI](https://tailwindui.com/), [Bootstrap](https://getbootstrap.com/), [Bulma](https://bulma.io/), or [Pure.css](https://purecss.io/) designed to cover a wide variety of needs. These frameworks follow various CSS architecture patterns or their own custom approach.
+
+Pros:
+
+- A lot of the work is done for us; less time spent creating custom styles.
+- Certain libraries provide us with the option to configure base styles and integrate with your build system.
+- Most have drop in React subsets that would allow us to utilize UI styling and corresponding markup as proper React components.
+- Most come with SASS/SCSS files to allow for configuration.
+
+Cons:
+
+- Won’t cover 100% of our design needs and possibly even less so than other options, which would lead to us needing to create custom styles or classes from scratch or _also_ use a utility framework.
+- Likely has the highest learning curve out of all considered options. This would require developers to know what already exists for use, likely causing constant documentation checks or searching for what’s available. An IDE extension could possibly ease this burden slightly, depending on the popularity of the chosen library.
+- Frameworks like Bootstrap are much heavier than other options. They'll almost certainly come with features/components that we will not make use of, though this could possibly be alleviated with a tool like PurgeCSS.
+- Introduces at least one more dependency to our stack. In fact, because some components would be shared and some would be project-specific it's likely that this dependency would exist in multiple areas of the repo. Even more dependencies would be required if we opt to use custom React components provided by the library or framework due to needing to install corresponding TypeScript definitions.
+- As some of these libraries include JavaScript to make components fully functional (e.g. accordions, tabs) we would likely need to have the security team vet it.
+
+### Option B - Use an existing utility framework
```

Comment:
This option seems like a library, not a framework.

Reply:
Haha @LZoog and I spent some time discussing this; I think her position was that it sounds like a library too. IMO these two terms have been muddied and are used so interchangeably these days it's tough to say exactly what's what. Your vote makes two, though, so I'm changing it!

Reply:
Heh, I did not mean to publish that, thought it was part of a draft review 🤦